src/Compilers/Core/Portable/CommandLine/CommonCommandLineArguments.cs

```diff
@@ -143,6 +143,12 @@ public abstract class CommandLineArguments
         public ImmutableArray<CommandLineSourceFile> AdditionalFiles { get; internal set; }
 
         /// <summary>
+        /// An ordered list of .editorconfig files that are consumed by analyzers. The order is significant, and if more than one .editorconfig
```

Comment:
😦 I think the ordering is going to come back to haunt us in the end. We're implementing a command line argument where the order of the arguments matters, but also the paths within the files themselves matters. It's going to work fine until someone calls csc on their own, at which point it will no longer make sense.

Also, note that the current implementation of `DiscoverEditorConfigFiles` is not compatible with this design, so the settings will not be correctly applied in all cases.